Integrations/QuinC/QuinC.yml

```diff
@@ -0,0 +1,99 @@
+commonfields:
+  id: Quinc
+  version: -1
+name: Quinc
+display: Quinc
+category: Forensics & Malware Analysis
+description: Use the Quinc integration to protect against and provide additional
+  visibility into phishing and other malicious email attacks.
+configuration:
+- display: Server URL (FQDN or IP address in X.X.X.X format)
+  name: server_name
+  defaultvalue: server_url
+  type: 0
+  required: true
+- display: The token is required to connect to Quinc.
+  name: Token
+  defaultvalue: 1861a70a-3287-4289-938b-bbab1446e2d2
+  type: 0
+  required: true
+- display: Scheme
+  name: Scheme
+  defaultvalue: http
+  type: 15
+  required: false
+  options:
+  - http
```

Comment:
no need for scheme parameters. include the whole url at server_url. `https://some_ip:port`